designs/metrics.md

```diff
@@ -72,10 +72,11 @@ Scheduling statistics consist of actions not representable by a single controlle
 termination and pod startup. The visualizations will be the same as those for individual controller performance. The
 following metrics will be instrumented to implement these visualizations:
 
-| Name                                       | Type    | Labels                                                          | Description                               |
-|--------------------------------------------|---------|-----------------------------------------------------------------|-------------------------------------------|
-| `karpenter_nodes_termination_time_seconds` | Summary | `provisioner`, `zone`, `arch`, `capacity_type`, `instance_type` | [Measurement Definitions](#measurment-definitions) |
-| `karpenter_pods_startup_time_seconds`      | Summary | `provisioner`, `zone`, `arch`, `capacity_type`, `instance_type` | [Measurement Definitions](#measurment-definitions) |
+| Name                                                   | Type      | Labels                                                          | Description                                        |
+|--------------------------------------------------------|-----------|-----------------------------------------------------------------|----------------------------------------------------|
+| `karpenter_nodes_termination_provisioner_time_seconds` | Summary   | `provisioner`                                                   | [Measurement Definitions](#measurment-definitions) |
+| `karpenter_nodes_termination_time_seconds`             | Summary   |                                                                 | [Measurement Definitions](#measurment-definitions) |
+| `karpenter_pods_startup_time_seconds`                  | Summary   | `provisioner`, `zone`, `arch`, `capacity_type`, `instance_type` | [Measurement Definitions](#measurment-definitions) |
```

Comment:
Like we discussed offline: let's keep these labels in, but when you add the metric in, filter down the labels to match the implementation.